lib/factor/factorQuadratic.js

```diff
@@ -246,12 +254,11 @@ function factorSumProductRule(node, symbol, aValue, bValue, cValue, negate) {
         const secondParen = Node.Creator.parenthesis(
           Node.Creator.operator('+', [firstCoeff, secondCoeff]));
 
-
         // create a node in the general factored form for expression
         let newNode = Node.Creator.operator('*', [firstParen, secondParen], true);
         if (gcd !== 1) {
           const gcdNode = Node.Creator.constant(gcd);
-          newNode = Node.Creator.operator('*', [gcdNode, newNode], true);
+          newNode = Node.Creator.operator('*', [gcdNode, firstParen, secondParen], true);
```

Comment:
since it's this instead, I think it should be an if / else for if gcd is 1 or not

also I think this line is too long now